### ISSUE TITLE
please implement tabbed windows

### ISSUE THREAD

**Issue by zaxebo1** [joewing/jwm#422]
when switching from fluxbox to JWM to minimize dependencies, everything goes well . But only one thing one misses and that is tabbed windows. If you implement it , (atleast as an optional compilation time enabling feature or may be a full feature), then JWM will be feature complete among the MIT licensed lightweight window manager users.

ref:
https://en.wikipedia.org/wiki/Comparison_of_X_window_managers

thanks in advance :-)

**Comment by technosaurus** [comment]
iirc jwm recently​ added group and list.  group combines tasks of the same class so that you get something similar to what you are asking for.  "list" (or is it "labeled"?) will display only the icon in the task bar (nice for touch screens)

**Comment by zaxebo1** [comment]
thanks for the information and quick reply. Actually i was using old version packaged with my linux distribution. I will download the latest version of jwm and will try that.

=======

Question:
Does your information also mean that at the wikipedia link  https://en.wikipedia.org/wiki/Comparison_of_X_window_managers  , we should update the "tabbed windows" column to "yes" for jwm ???

**Comment by technosaurus** [comment]
@zaxebo1 Not exactly.  The window itself does not have the tabs when "grouped", but all instances are grouped together in one place in the taskbar.   I haven't used grouping in a while except to test it, but if it doesn't already exist it would probably be a smallish patch to duplicate  the taskbar functionality into the title bar (maybe by right clicking the icon or title on the title bar) which would be good for things like [surf web browser](https://surf.suckless.org/) to replace the [tabbed](https://tools.suckless.org/tabbed/) utility.
 
 Since jwm doesn't support drag and drop, [fluxbox's dragging window to another windows tab area](https://web.archive.org/web/20111007025050/http://old.fluxbox.org:80/features/tabs.php) would be more difficult.

**Comment by mmetak** [comment]
Hello. Any new progress on this issue? Recently started trying out jwm and I really like it and one feature that I miss the most from i3 is tabbed windows. It really does make things so much better.

**Comment by JBHoren** [comment]
> Hello. Any new progress on this issue?

I was looking at PekWM and was impressed with its "window grouping" feature -- it's a capability that would enhance my long-time (now more than a dozen years) use and enjoyment of JWM. Please?
